### DISCUSSION TITLE
[Feature request] Strip the banner when reply

### DISCUSSION THREAD

**Discussion by http403** [anonaddy/anonaddy#351] [Feature Requests]
The banner is quite useful but it takes quite some space and it appened every single time when the sender replies my mail, thus building up more and more useless infomation to the sender. Of course I can do it manually everytime but it's way less error prone (aka forgotten) when automated.

**Comment by willbrowningme** [comment]
Thanks, there is currently an issue open for this https://github.com/anonaddy/anonaddy/issues/253.

I'll see if I can add it when I get time.

**Reply by http403** [reply]
Thanks. I didn't notice that as I only searched the discussion, not the issue. My bad.

**Reply by willbrowningme** [reply]
This should be working now, please give it a try and let me know if everything is as expected. (It will only remove the banner from newly forwarded messages starting from now).

**Reply by http403** [reply]
Sorry for the late reply. I can confirm that both plain and HTML mail work correctly. Thanks for the work!
